apps/scratch/src/instrument.ts

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from "@sentry/react";
+import { sentryDsn } from "./utilities/constants";
+
+Sentry.init({
+  dsn: sentryDsn,
+  integrations: [
+    Sentry.browserTracingIntegration(),
+    Sentry.replayIntegration(),
+  ],
+  // Tracing
+  tracesSampleRate: 1.0, //  Capture 100% of the transactions
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  tracePropagationTargets: ["localhost"],
+  // Session Replay
+  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
```

Comment:
The comment makes sense.
Can we have two configs, one for dev and one for prod?

Reply:
I mean we can add as many environment variables as we like, yes. Do you see any other option you would like as a environment variable?